torch/csrc/jit/passes/peephole.cpp

```diff
@@ -40,7 +40,15 @@ void PeepholeOptimize(std::shared_ptr<Graph>& graph) {
         }
         break;
     }
+
+    for (Block * sub_block : n->blocks()) {
+      PeepholeOptimize(sub_block);
```

Comment:
Similar comment here to fuser: if peephole optimization previously assumed everything was in the same block, those checks need to be added now.